README.md

```diff
@@ -1,3 +1,24 @@
 # Jumpstart Templates
 
-This repo contains jumpstart templates, 1 directory per package.
+Welcome to the Jumpstart Templates repository! This repository contains a collection of templates for various frameworks and languages to help you kickstart your projects with ease using the Jumpstart tool.
+
+
+## Contribution Guidelines
+We welcome contributions to the Jumpstart Templates repository! Follow these guidelines to contribute:
+
+1. Fork the repository to your GitHub account.
+
```

Comment:
```suggestion
```
Lets get rid of this line, its causing un-necessary line break in list items, github bug 🤔 ?

Reply:
daym how did that get in : | ... fixed